Symfony/CS/Tests/Fixer/Symfony/PhpUnitDedicateAssertFixerTest.php

```diff
@@ -0,0 +1,180 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Tests\Fixer\Symfony;
+
+use Symfony\CS\Tests\Fixer\AbstractFixerTestBase;
+
+/**
+ * @author SpacePossum
+ *
+ * @internal
+ */
+final class PhpUnitDedicateAssertFixerTest extends AbstractFixerTestBase
+{
+    /**
+     * @dataProvider provideInternalTypeMethods
+     */
+    public function testInternalTypeMethods($expected, $input = null)
+    {
+        $this->makeTest($expected, $input);
+    }
+
+    public function provideInternalTypeMethods()
+    {
+        $cases = array();
+        foreach (array('array', 'bool', 'float', 'int', 'string') as $type) {
+            $cases[] = array(
+                sprintf('<?php $this->assertInternalType(\'%s\', $a);', $type),
+                sprintf('<?php $this->assertTrue(is_%s($a));', $type),
+            );
+
+            $cases[] = array(
+                sprintf('<?php $this->assertNotInternalType(\'%s\', $a);', $type),
+                sprintf('<?php $this->assertFalse(is_%s($a));', $type),
+            );
+        }
+
+        $cases[] = array(
+            '<?php $this->assertInternalType(\'float\', $a, "my message");',
```

Comment:
I think it don't need to run after it, just handle the aliased function here.
We combine fixers in correct order to do sth that is not possible by one fixer itself. Here it's not that case.